index.js

```diff
@@ -1,24 +1,70 @@
 const Router = require('express').Router;
 
-module.exports = (options) => {
+/**
+ * A helper for `createNamedAsyncMiddleware`
+ * If the function already has a name, we'll just use that and tack
+ * `Async` on the end. If the function doesn't have a name we'll
+ * use the path for the target handler plus the index of the current
+ * middleware.
+ */
+function getFunctionName(fun, handlerPath, middlewareIndex) {
+  if (fun.name && fun.name !== 'anonymous') {
```

Comment:
please check for empty string too

I ran this on node and got empty string instead of anon: `console.log((() => {}).name)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name

Reply:
It's already checked?

Reply:
Empty strings are falsy!